lib/src/platform/native/media_stream_track.dart

```diff
@@ -1,6 +1,8 @@
 import 'dart:async';
+import 'package:flutter/foundation.dart';
 
 import 'package:flutter/services.dart';
+import 'package:medea_flutter_webrtc/src/platform/native/medea_flutter_webrtc_method_channel.dart';
 
 import '/medea_flutter_webrtc.dart';
```

Comment:
This files has only relative imports. Why would you break this consistency? Is there a reason to do an absolute package import instead of relative absolute once as below? If yes, please, write the reason down in this conversation.